src/App.tsx

```diff
@@ -54,6 +54,7 @@ import NurseryDetails from './components/Nursery';
 import InventoryCreate from './components/Inventory/InventoryCreate';
 import InventoryView from './components/Inventory/InventoryView';
 import PlantingSites from './components/PlantingSites';
+import CreatePlantingSite from './components/PlantingSites/PlantingSiteCreate';
```

Comment:
Suggestion, we could export all components from plantingsites/index.tsx
sort of like what we did for accessions2 (line 47 here) - we missed doing this for inventory. Generally cleaner/organized this way?